### PR TITLE
Add miscProperties and originalSensitiveData to the DWCA export

### PR DESCRIPTION
**Pull request by sadeghim**

### CODE REVIEW

File: src/main/scala/au/org/ala/biocache/processor/LocationProcessor.scala [hunk 1]
```diff
@@ -161,6 +164,24 @@ class LocationProcessor extends Processor {
     }
   }
```
Comment (djtfmartin) [inline src/main/scala/au/org/ala/biocache/processor/LocationProcessor.scala:166]: Not sure why this is part of this PR as it seems unrelated.

Comment (sadeghim) [reply src/main/scala/au/org/ala/biocache/processor/LocationProcessor.scala:166]: Yes, you are right. There are 14 PRs waiting in this repo and I don't know if it's possible to just choose my commits for this PR. What's your suggestion?

**Comment by sadeghim** [comment]
> There seems to be additional logic in this PR that isnt associated with the DwCA export. Perhaps its for another PR ?
Yes, you are right. There are 14 PRs waiting in this repo and I don't know if it's possible to just choose my commits for this PR. What's your suggestion?

File: src/main/scala/au/org/ala/biocache/processor/LocationProcessor.scala (hunk 1 above)
Comment (sadeghim) [reply src/main/scala/au/org/ala/biocache/processor/LocationProcessor.scala:166]: Refer to the comment below.

**Comment by djtfmartin** [comment]
I'll mark as approved. I guess for future work it might be cleaner if we start doing feature branches targetted at `develop` tied to specific issues as we have started doing in `gbif/pipelines`.
